wpeverest/user-registration
# Enhance/real time password matching

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds the feature of real time password matching. It checks if the user provided passwords (i.e. password and confirm password) matches or not and shows error message if they don't match. It also doesn't allow the form to be submitted, if the passwords are different.




### How to test the changes in this Pull Request:

1. Add the `Password` and `Confirm Password` fields
2. Go to the frontend
3. Check if it's working properly
4. Also test it in the `Change Password` form

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Real time password matching
